app/Events/GameEvent.php

```diff
@@ -14,6 +14,7 @@ abstract class GameEvent implements ShouldBroadcastNow
 
     protected $gameId;
     public $message;
+    public $timeLeft;
```

Comment:
Moet $timeLeft wel in GameEvent zitten als het alleen in ResumeGameEvent een waarde krijgt?